### PR TITLE
Update ensembling.md

### PR DESCRIPTION
**Pull request by wesmare**
Clarified weak supervision, much more spelled out example

### CODE REVIEW

**Comment by trigaten** [comment]
```


SyntaxError: /vercel/path0/docs/reliability/ensembling.md: Expected corresponding JSX closing tag for <div>. (133:0)
--
19:48:36.558 | 131 \| <div>Question: "What is the capital of France?"</div>
19:48:36.558 | 132 \| <div>Answer: "Paris"</div>

```

**Comment by wesmare** [comment]
If pulling SyntaxErrors was cool, I’d be Miles Davis. Will fix and commit.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: docs/reliability/ensembling.md [hunk 1]
```diff
@@ -151,49 +151,75 @@ I will not cover here. Read section 3.3 of the paper for more details(@li2022adv
 
 In the same multi-input style as DiVeRSe prompting, *Ask Me Anything (AMA)* prompting(@arora2022ama) is an approach that uses multiple imperfect prompts—rather than one painstakingly crafted prompt.
 
-The *AMA* prompting strategy combines the results from the different prompts by utilizing weak supervision to aggregate their output answers(@arora2022ama) to get an effective best answer. 
+The *AMA* prompting strategy combinds the results from the multiple different prompts by utilizing weak supervision to aggregate their output answers(@arora2022ama) to get an effective best answer.
```
Comment (trigaten) [inline docs/reliability/ensembling.md:154]: combines?

**Comment by wesmare** [comment]
Combines…that’s the one.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wesmare** [comment]
Removed some confusing language about "Combining" prompt outputs. should be more clear now

**Comment by wesmare** [comment]
@trigaten let me know if you need/want any other changes to merge and post

**Comment by trigaten** [comment]
Im reading the paper and there is a lot more to this method. How much of it did you read?

**Pull request merged**